### ISSUE TITLE
Support for `DISCONNECT ALL`

### ISSUE THREAD

**Issue by GitMensch** [mridoni/gixsql#89]
Just dropping it here as it came up with ESQL once and while I personally don't need it currently it would be a good thing to have implemented and added to the testsuite:  `DISCONNECT ALL`.

Would it be handled in the driver (= all open connections stored, then closed at once) or by the underlying library?

~~~
::cobol
           EXEC SQL
               DISCONNECT ALL
           END-EXEC.
~~~

> unsupported SQL: `DISCONNECT ALL`

[IBM docs](https://www.ibm.com/support/knowledgecenter/SSGU8G_11.50.0/com.ibm.esqlc.doc/sii-11-58157.htm)
[Oracle docs](https://docs.oracle.com/cd/B19306_01/server.102/b14357/ch12052.htm)

_Originally posted by @GitMensch in https://github.com/mridoni/gixsql/issues/72#issuecomment-1179534596_

**Comment by mridoni** [comment]
Implemented in the internal repository, will be in v1.0.17. Fixed a few memory leaks along the way.

**Comment by GitMensch** [comment]
According to the release notes: solved.
